client/jetpack-cloud/sections/partner-portal/license-product-card/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { ReactElement } from 'react';
+import './style.scss';
+
+export default function LicenseProductCard( props: any ): ReactElement {
+	const {
+		product: { label },
+		orderIndex,
+	} = props;
+	const productTitle = label.replace( 'Jetpack ', '' ).replace( '(', '' ).replace( ')', '' );
+
+	function handleKeyDown( e: any ) {
+		if ( 32 === e.keyCode ) {
```

Comment:
If we support "keyDown", should we perhaps listen for key `13` (enter) as well since that's how most people submit forms and we're mimicking that behaviour?

We don't seem to have any styling to indicate that a box is "selected". Example of how to support it:

```
.license-product-card:focus .license-product-card__inner {
    border-color: green;
    box-shadow: 0 5px 10px black;
}
```